call-retrain.py

```diff
@@ -4,126 +4,17 @@
-                "type": "execution",
-                "template": {
-                    "environment": "017d329a-e56f-d451-040b-ccb895513c40",
-                    "commit": "0aadea924a866f2fd95b5ea304c2a8f7fd0cc189",
-                    "step": "train_model",
-                    "image": "ultralytics/yolov5",
-                    "command": "pip install valohai-utils\npython train.py {parameters}",
-                    "inputs": {
-                        "train": ["dataset://drift-demo-ships-aerial/dev_train"],
-                        "test": ["dataset://drift-demo-ships-aerial/dev_test"],
-                        "valid": ["dataset://drift-demo-ships-aerial/dev_valid"],
-                    },
-                    "parameters": {
-                        "yolo_model_name": "yolov8x.pt",
-                        "epochs": 3,
-                        "batch_size": 8,
-                        "image_size": 768,
-                        "optimizer": "SGD",
-                        "seed": 42,
-                        "project": "/valohai/outputs/",
-                    },
-                    "runtime_config": {},
-                    "runtime_config_preset": "",
-                    "inherit_environment_variables": True,
-                    "environment_variable_groups": [],
-                    "tags": [],
-                    "time_limit": 0,
-                    "environment_variables": {},
-                },
-                "on_error": "stop-all",
-            },
-            {
-                "name": "evaluation",
-                "type": "execution",
-                "template": {
-                    "environment": "017d329a-e56f-d451-040b-ccb895513c40",
-                    "commit": "0aadea924a866f2fd95b5ea304c2a8f7fd0cc189",
-                    "step": "evaluation",
-                    "image": "ultralytics/yolov5",
-                    "command": "pip install valohai-utils\npython evaluation.py {parameters}",
-                    "inputs": {
-                        "model": ["datum://model-current-best"],
-                        "data_yaml": ["datum://data_yaml"],
-                        "valid": ["dataset://drift-demo-ships-aerial/dev_valid"],
-                        "test": ["dataset://drift-demo-ships-aerial/dev_test"],
-                    },
-                    "parameters": {},
-                    "runtime_config": {},
-                    "runtime_config_preset": "",
-                    "inherit_environment_variables": True,
-                    "environment_variable_groups": [],
-                    "tags": [],
-                    "time_limit": 0,
-                    "environment_variables": {},
-                },
-                "on_error": "stop-all",
-            },
-        ],
-        "project": "018c8779-9475-09e1-d481-e295ab4de428",
-        "tags": [],
-        "parameters": {},
-        "title": "train-val-pipeline",
-    },
+        "project": "018d30f4-d4fc-91f6-f67b-6ab28f72f3f8",
```

Comment:
This should presumably use [`VH_PROJECT_ID`](https://docs.valohai.com/hc/en-us/articles/18704309037073-System-Environment-Variables)?

Reply:
Didn’t know that that we have it